isos/appliance/nat-setup

```diff
@@ -81,6 +81,8 @@ COMMIT
 -A INPUT -p tcp -m tcp --dport 2378 -j ACCEPT
 -A INPUT -p tcp -m tcp --dport 2380 -j ACCEPT
 
+-A INPUT -p tcp -m tcp --dport 6060 -j ACCEPT
```

Comment:
it actually would be awesome to have a comment for each line to explain for which service that port is being opened.

Reply:
Agreed - I don't think it can be done inline, but might be wrong. Will add it inline if I can, out side the block if not.